tests/unit/deployment/test_job.py

```diff
@@ -60,7 +60,7 @@ def test_deploy_new_job(oracle_recon_config):
     product_info = ProductInfo.from_class(RemorphConfigs)
     name = "Recon Job"
     job_deployer = JobDeployment(workspace_client, installation, install_state, product_info)
-    job_deployer.deploy_recon_job(name, oracle_recon_config)
+    job_deployer.deploy_recon_job(name, oracle_recon_config, "remorph-0.5.0-py3-none-any.whl")
```

Comment:
this will break when we release new versions.

Reply:
I have tested with the latest release, it's not breaking. 
Here we are creating mocked objects and asserting `jobs.create.assert_called_once()`. 
So, I don't think it will fail in new versions. Please advise.